lib/src/widget/built_in/built_in.dart

```diff
@@ -101,6 +92,15 @@ class BuiltInContent extends StatelessWidget {
       );
     }
 
-    return content;
+    return buildTitle(context);
+  }
+
+  Text buildTitle(BuildContext context) {
```

Comment:
I know that builder methods are very common and popular in many languages. However, I used local variable, `Widget content`, pattern because it used commonly in flutter code base.

Check out [this link](https://github.com/flutter/flutter/blob/f468f3366c26a5092eb964a230ce7892fda8f2f8/packages/flutter/lib/src/widgets/container.dart#L380).

Do you believe `builderMethods' is 
better in performance or readibility?

Reply:
Wow I didn't know this is a common pattern in flutter! In my point of view using builder is more readable, I guess they won't have that much difference in performance, So we can keep using local variables. Thanks.